components/PageLoadProgress/PageLoadProgress.tsx

```diff
@@ -0,0 +1,7 @@
+import NextNProgress from "nextjs-progressbar";
+import { FCC } from "types/IReact";
+
+export const PageLoadProgress: FCC = () => {
+	const progressBarColor = "#FF5959";
```

Comment:
call the `useTheme()` hook to get the `primary` color from `palette`

`useTheme()` is used multiple times in other files in this project. you can refer those to learn how get primary color